Sources/ZcashLightClientKit/Rust/ZcashRustBackend.swift

```diff
@@ -24,9 +24,51 @@ enum RustLogging: String {
     case trace
 }
 
+/// A description of the policy that is used to determine what notes are available for spending,
+/// based upon the number of confirmations (the number of blocks in the chain since and including
+/// the block in which a note was produced.)
+///
+/// See [`ZIP 315`] for details including the definitions of "trusted" and "untrusted" notes.
+///
+/// # Note
+///
+/// `trusted` and `untrusted` are both meant to be non-zero values.
+/// `0` will be treated as a request for a default value.
+///
+/// [`ZIP 315`]: https://zips.z.cash/zip-0315
+public struct ConfirmationsPolicy {
+    /// NonZero, zero for default
+    let trusted: UInt32
+    /// NonZero, zero for default; if this is set to zero, `trusted` must also be set to zero
+    let untrusted: UInt32
+    let allowZeroConfShielding: Bool
+    
+    init(trusted: UInt32 = 3, untrusted: UInt32 = 10, allowZeroConfShielding: Bool = true) {
+        self.trusted = trusted
+        self.untrusted = untrusted
+        self.allowZeroConfShielding = allowZeroConfShielding
+    }
+    
+    public static func defaultTransferPolicy() -> Self {
+        ConfirmationsPolicy.init()
+    }
+    
+    public static func defaultShieldingPolicy() -> Self {
+        ConfirmationsPolicy.init(trusted: 1, untrusted: 1, allowZeroConfShielding: true)
+    }
```

Comment:
```suggestion
    /// Returns a default ConfirmationPolicy suitable for shielding transactions.
    public static func defaultShieldingPolicy() -> Self {
        .init(trusted: 1, untrusted: 1, allowZeroConfShielding: true)
    }
```
This change is in #1592 but it should be here.